glide.yaml

```diff
@@ -289,3 +289,4 @@ import:
   version: v1.0
 - package: github.com/klauspost/cpuid
   version: v1.0
+- package: github.com/jonboulle/clockwork
```

Comment:
Please split glide changes in their own commits (best would be one for YAML and one for vendoring).